arguments-optional.js

```diff
@@ -0,0 +1,16 @@
+/**
+ *https://learn.freecodecamp.org/javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/arguments-optional/ 
+ *
+ * @param {*} a
+ * @param {*} b
+ * @returns {number|undefined} sum of a and b or undefined
+ */
+function addTogether(a, b) {
+  if (typeof(a, b) === "number") return a + b;
+
+  if (arguments.length === 1 && typeof(a) === "number") 
```

Comment:
I still don't think you need the `arguments` object, but that's okay.

Reply:
I did try it without it and it didn't work